gremlin-server/Dockerfile

```diff
@@ -15,15 +15,16 @@
 # specific language governing permissions and limitations
 # under the License.
 
-FROM adoptopenjdk/openjdk11:alpine-slim
+FROM alpine
```

Comment:
What is the image size comparing to `adoptopenjdk/openjdk11:alpine-slim`?

Reply:
No change.
Final built image size is 373MB. The tinkerpop/gremlin-server image on dockerhub is 375MB.